### PR TITLE
QA: Review Branch and Synced Sandboxes Branch cards

### PR DESCRIPTION
**Pull request by lenasmayor**
Closes XTD-213 and XTD-238

1. Remove cloud beta tag from `Branch` cards.
(before)
Screen Shot 2022-10-05 at 13 02 58

(after)
Screen Shot 2022-10-05 at 12 51 25

2. Review `Synced Sandbox branch` card. 
(before) 
Screen Shot 2022-10-05 at 13 02 31

(after)
Screen Shot 2022-10-05 at 12 59 06

3. Synced Sandboxes empty state: Removed `New` card and changed text
(before)
Screen Shot 2022-10-05 at 13 01 55

(after)
Screen Shot 2022-10-05 at 12 59 41

4. Correct paddingX of repository cards
(before)
Screen Shot 2022-10-05 at 13 01 18

(after)
Screen Shot 2022-10-05 at 13 00 36

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6978/commits/96c69e6ea5c6781e62850434ca881b6611767ade is at https://pr6978.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx [hunk 1]
```diff
@@ -54,16 +52,13 @@ export const BranchCard: React.FC<BranchProps> = ({
...
-          paddingY: 10,
+          paddingY: 11,
           position: 'relative',
         })}
         align="center"
         justify="center"
       >
-        <Icon color="#808080" name="branch" size={40} />
-        <Element css={{ position: 'absolute', top: 6, right: 6 }}>
-          <CloudBetaBadge />
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx:65]: We're not using the cloud beta badge anywhere anymore?

**Review by tristandubbeld — APPROVED** [review]
All looks good to me, but I have one question regarding the cloud beta badge.

File: packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx (hunk 1 above)
Comment (alexnm) [reply packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx:65]: I would still show it on the branch cards for the recent page, but we can merge this as it is as it fixes a lot of design insconsistencies and address it later?

**Review by alexnm — APPROVED** [review]

**Pull request merged**